src/components/KeyboardSpacer/components/index.js

```diff
@@ -0,0 +1,121 @@
+import React, {PureComponent} from 'react';
+import {
+    Keyboard,
+    LayoutAnimation,
+    View,
+    Dimensions,
+    ViewPropTypes,
+    Platform,
+    StyleSheet,
```

Comment:
NAB, let's put these in alphabetical order.

Reply:
sure